_partials/vscode_extensions.md

```diff
@@ -2,7 +2,20 @@
 
 ### Installation
 
-Let's install some useful extensions to VS Code.
+Let's install some useful extensions to VS Code. 
+
+- [Sublime Text Keymap and Settings Importer](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings)
+- [VSCode Great Icons](https://marketplace.visualstudio.com/items?itemName=emmanuelbeziat.vscode-great-icons)
+- [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare)
+- [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+- [Python Indent](https://marketplace.visualstudio.com/items?itemName=KevinRose.vsc-python-indent)
+- [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)
+- [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
+
+
+☝️ If you are on **Windows**, install them **manually** one-by-one from within VS code (Ctrl-Shift-X)
+
+If you are on **Mac** or **Linux**, you can install them by running the following command from your terminal: 
```

Comment:
A nice hacks is using the `details/summary` HTML pattern to propose alternative setup if you don't want to write a new partial. It should be better to have a different partial for WSL users because manual installation requires some screenshots. Let's improve that later

Reply:
A possible option is to use delimiters to separate content :

```
$WINDOWS_START
windows content
$WINDOWS_END
$MAC_START
mac content
$MAC_END
$LINUX_START
linux content
$LINUX_END
```

Reply:
This does not support yet combinations (mac + linux)

Reply:
Sorry for the late answer. 

I don't get why you want to separate the way you install extensions for Windows on one side and macOS & Linux on the other side since the list of extensions is the same.

The command `code --install-extension NAME-OF-THE-EXTENSION` is working on every terminal on Windows, whether WSL or CMD. 

For example, for the web setup, here is how we install extension from the command line : https://github.com/lewagon/setup/blob/master/windows.md#vs-code-extensions

Am I missing something? :thinking: